docs/sections/use-case-requirements.html

```diff
@@ -10,30 +10,230 @@ <h3>Use Cases and Requirements</h3>
+</pre>
+
+ <p>
+2. Review the <code>serviceEndpoint</code>, <code>assertionMethod</code>, and <code>authentication</code> sections of the DID document.
+ </p>
+<section>
+  <h5>service</h5>
+
+
+<p>
+The serivce block provides a means of communicating or interacting with the DID subject or associated entities via one or more service endpoints.
+</p>
+
+  <p>
+    See <a href="https://www.w3.org/TR/did-core/#dfn-service">service</a>.
+  </p>
+
+<p>
+This entry MUST be present.
+</p>
+<p>
+This entry MUST have an element of type <code>TraceabilityAPI</code> with a <code>serviceEndpoint</code> URL for a VC-API that supports presentation exchange.
+</p>
+
+  <p>For example:</p>
+
+<pre class="example">
+"service": [
+  {
+    "id": "did:web:platform.example:organization:123#traceability-api",
+    "type": "TraceabilityAPI",
+    "serviceEndpoint": "https://platform.example/organization/123"
+  }
+]
+</pre>
+
+<p>This definition implies the following endpoints:</p>
+
+<pre class="example">
+https://platform.example/organization/123/credentials/issue
+https://platform.example/organization/123/credentials/verify
+https://platform.example/organization/123/presentations/prove
+https://platform.example/organization/123/presentations/verify
+https://platform.example/organization/123/presentations/available
+https://platform.example/organization/123/presentations/submissions
+https://platform.example/organization/123/presentations
+</pre>
+
+<p>
+These endpoints require the application to be authenticated:
+</p>
+
+<pre class="example">
+https://platform.example/organization/123/credentials/issue
+https://platform.example/organization/123/presentations/prove
+</pre>
+
+<p>
+These endpoints do not require the application to be authenticated:
+</p>
+
+<pre class="example">
+https://platform.example/organization/123/presentations/available
+https://platform.example/organization/123/presentations/submissions
+</pre>
+
+</section>
+
+<section>
+  <h5>assertionMethod</h5>
+<p>
+The set of supported <code>assertionMethod</code> DID URLs for the organization.
+</p>
+
+<p>
+See <a href="https://www.w3.org/TR/did-core/#assertion">assertionMethod</a>.
+</p>
+</section>
+
+<section>
+  <h5>authentication</h5>
+<p>
+The set of supported <code>authentication</code> DID URLs for the organization.
+</p>
+
+<p>
+See <a href="https://www.w3.org/TR/did-core/#authentication">authentication</a>.
+</p>
+</section>
+
+ <p>
+  The <code>didDocument</code> MAY contain a <code>verificationMethod</code> section,
+  which MAY be used to support <code>did:web</code> based verification relationships.
+ </p>
+<p>
+  The <code>didDocument</code> MUST NOT contain <code>verificationMethods</code> where the controller is different from the DID Subject.
+</p>
+
+<p>
+  All <code>Ed25519VerificationKey2018</code> types support <code>Ed25519Siganture2018</code>.
+</p>
+
+<p>
+  All <code>JsonWebKey2020</code> types support <code>JsonWebSignature2020</code> AND <code>VC-JWT</code>.
```

Comment:
do we need to have `AND` here? 
aren't most implementations of JWS2020 LD-Proof based rather than JWT based?

Reply:
This language is intentionally inclusive, also these are 2 separate things... happy to explain on a call.

Reply:
explained on the call, JWKs can be used to verify multiple proof formats